.env

```diff
@@ -0,0 +1,5 @@
+DB_HOST=localhost
```

Comment:
Given these env vars, I was not able to fully get the project running. I ran into an application runtime issue:

```
Error loading data into test2db: could not connect to server: Connection refused Is the server running on host "localhost" (::1) and accepting TCP/IP connections on port 5432? could not connect to server: Connection refused Is the server running on host "localhost" (fe80::1) and accepting TCP/IP connections on port 5432? could not connect to server: Connection refused Is the server running on host "localhost" (127.0.0.1) and accepting TCP/IP connections on port 5432? 
```

What am I missing here?

Reply:
The values mentioned in the .env file pertain to my local test database that I had setup to run and validate this application. As mentioned in step 2 https://github.com/vishket/gbapp/tree/vs_solution#pre-requisites , it requires the user to enter his own Database values. Please let me know if you have any questions or need clarification on this?